client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/buffer/BufferPacker.java

```diff
@@ -216,6 +217,16 @@ public BufferRecycler getRecycler() {
       return buffer.getRecycler();
     }
 
+    // Flink 1.18.0
+    // [FLINK-32549][network] Tiered storage memory manager supports ownership transfer for buffers
+    public void setRecycler(BufferRecycler bufferRecycler) {
```

Comment:
Flink 1.18.0
[[FLINK-32549](https://issues.apache.org/jira/browse/FLINK-32549)][network] Tiered storage memory manager supports ownership transfer for buffers